app/pb_kits/playbook/pb_flex/docs/_description.md

```diff
@@ -0,0 +1 @@
+Used the same way flex box is used.
```

Comment:
You might expand this a bit. We use this to build most of the complex interfaces